cni/network/network.go

```diff
@@ -479,18 +436,12 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 			zap.String("pod", k8sPodName),
 			zap.Any("IPs", cniResult.IPs),
 			zap.Error(log.NewErrorWithoutStackTrace(err)))
+
+		telemetry.SendEvent(fmt.Sprintf("ADD command completed with [ipamAddResult]: %s [epInfos]: %s [error]: %v ", ipamAddResult.PrettyString(), network.FormatStructPointers(epInfos), err))
```

Comment:
Per one of your previous comments, `telemetry.SendEvent` will both write to `azure-vnet.log` as well as send to telemetry service. Aren't we then writing to the log file twice here if `logger` also writes to the file? Can we keep this PR to **only** a pure refactor without introducing new side effects?

Reply:
yes you are correct, I've removed the duplicate logger statement in the telemetry client